lib/Pusher.php

```diff
@@ -330,6 +352,7 @@ public function trigger( $channels, $event, $data, $socket_id = null, $debug = f
 
 		if ( $socket_id !== null )
 		{
+			$this->validate_socket_id( $socket_id );
```

Comment:
OK, but that will just move the if statement into the `validate_socket_id` function.